gobblin-salesforce/src/main/java/org/apache/gobblin/salesforce/SalesforceExtractor.java

```diff
@@ -779,7 +880,7 @@ public boolean bulkApiLogin() throws Exception {
       BatchInfoList batchInfoList = this.bulkConnection.getBatchInfoList(this.bulkJob.getId());
```

Comment:
Remove this?

Reply:
>>       BatchInfoList batchInfoList = this.bulkConnection.getBatchInfoList(this.bulkJob.getId());
I try not to change the existing code in **getQueryResultIds**. And this line is necessary. **getBatchInfoList** function's return value is **BatchIdAndResultId** which means a list of resultIds with corresponded batch Ids.
One bulkJobId could have multiple results even though it is not a pk-chunking.

I am not use **getQueryResultIds** for pkchunking. If we want to do further code refactoring, let's do it in another ticket.